integration/README.md

```diff
@@ -0,0 +1,704 @@
+# Running Snap in various environments
```

Comment:
Removing "integration" directory would cause conflict of two directories with the same name "src". I close this PR and push the changes to integration branch as you suggest.